JavaOpdracht6/familyTree/src/main/java/Main.java

```diff
@@ -19,24 +23,25 @@ public static void main(String[] args) {
 
         System.out.println(martin.getMiddleName());
 
-        irene.addParents(hanneke, martin);
+        irene.addParents(hanneke, martin, irene);
         System.out.println(irene.getMother().getName());
 
-        // Hier komt een handler uit (Person@372f7a8d). Hoe kan ik ervoor zorgen dat de namen van de kinderen uitgeprint worden?
+        // Hier komt een handler uit (Person@372f7a8d). Hoe kan ik ervoor zorgen dat de namen van de kinderen uitgeprint worden? (dezelfde vraag bij de methode addPet onderaan in Main)
```

Comment:
children is een List, dus:
 System.out.println(joris.getChildren().get(0).getName());